boards.txt

```diff
@@ -818,3 +818,50 @@ ng_beacon.menu.softdevice.s130.softdeviceversion=2.0.1
 ng_beacon.menu.softdevice.s130.upload.maximum_size=151552
 ng_beacon.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 ng_beacon.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_xxaa.ld
+
+
+Sinobit.name=Sino:bit
+
+Sinobit.vid.0=0x0d28
```

Comment:
Since we don't have a sino:bit to test with, are these the correct USB id's (same as micro:bit)?

Reply:
Yes, It's the MBED, cmsis-DAP 0d28:0204